components/feature/downloads/src/main/res/xml/feature_downloads_file_paths.xml

```diff
@@ -4,4 +4,7 @@
    - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
 <paths>
     <external-path name="Download" path="." />
+
+    <!-- Offer access only to files under Context.getCacheDir()/media_share_cache/ -->
+    <cache-path name="mediaShareCache" path="mozac_share_cache"/>
```

Comment:
This is an important change that adds support for exposing private data.
I expect no security issues though since we will only share from a specific location.
Using the internal cache directory simplifies the logic of the feature - which only has to know of a single directory if which it is in full control and in our case helps with ensure a smooth cleanup of old files.